engine/worker/cmd_tmpl.go

```diff
@@ -98,6 +108,14 @@ func tmplCmd(w *currentWorker) func(cmd *cobra.Command, args []string) {
 	}
 }
 
+func getAbsoluteDir(arg string, currentDir string) string {
```

Comment:
With early return:
```go
func getAbsoluteDir(arg string, currentDir string) string {
	if strings.HasSuffix(arg, string(filepath.Separator)) {
		return arg
	}
	return filepath.Join(currentDir, arg)
}
```

--HasPrefix --> it's HasPrefix, no? :)

Reply:
You are right ;)